buttersink/SSHStore.py

```diff
@@ -343,7 +343,7 @@ def _open(self):
             self._mode,
             self._directory
         ]
-        logger.debug("Connecting with: %s", cmd)
+        logger.debug("Connecting with: %s" % cmd)
```

Comment:
The previous way of logging is actually preferred, because it can avoid some computation, if the log is not needed.

In the previous code if logging level is set to anything above debug, no string manipulation will be done once we pass this line. In the newer version of the code, strings will be modified before the function is called, which in some cases (longer messages and such) can be a noticeable performance degradation.

Also you can see in the docs that the using string parameters as arguments is preferred:
first code example here https://docs.python.org/2/library/logging.html